src/cli/restore/restore.go

```diff
@@ -34,15 +34,41 @@ func AddCommands(cmd *cobra.Command) {
 
 const restoreCommand = "restore"
 
+//nolint:lll
+const restoreCommandExamples = `# Restore emails to their original location
+corso restore exchange \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--destination '/'
```

Comment:
We should have the examples be more realistic. I don't think restore of the full service to the original location is such an example. 

Would recommend restore of a specific item or folder instead.

Reply:
done, added `--email-folder '/inbox'` as the selector.